src/verify_ce_config.py

```diff
@@ -59,19 +59,22 @@ def main():
         try:
             config_val = htcondor.param[attr]
         except KeyError:
-            error("Missing required %s configuration value" % attr)
+            warn("Missing %s configuration value. " % attr +
+                 "Can be safely ignored if using 'JOB_ROUTER_ROUTE_*' ClassAd transforms.")
 
         # store the ads (iterating through ClassAdStringIterator consumes them)
         try:
             parsed_jr_ads[attr] = list(classad.parseAds(config_val))
         except ValueError:
             # We shouldn't ever get here since classad.parseAds() only raises ValueError when it's given
             # non-string/non-file output and htcondor.param shouldn't contain such values
-            error("Failed to parse %s configuration value" % attr)
+            warn("Failed to parse %s configuration value. " % attr +
+                 "Can be safely ignored if using 'JOB_ROUTER_ROUTE_*' ClassAd transforms.")
 
         # If JRD or JRE can't be parsed, the job router can't function
         if not parsed_jr_ads[attr]:
-            error("Could not read %s in the HTCondor-CE configuration." % attr)
+            warn("Could not read %s in the HTCondor-CE configuration. " % attr +
```

Comment:
Is the admin going to get multiple warnings for the same thing?

Reply:
Yes, once each for an unparseable JRE and JRD. The admin will really have to go out of their way to unset the latter, though.